docs/conf.py

```diff
@@ -89,10 +92,48 @@
 napoleon_use_admonition_for_references = False
 napoleon_use_ivar = False
 napoleon_use_param = True
-napoleon_use_rtype = True
+napoleon_use_rtype = False
 
 numfig = True
 
+# numpydoc
+
+
+# find whether the object is part of plenoptic
+def is_part_of_plenoptic(obj):
+    try:
+        return obj.__module__.startswith("plenoptic")
+    except AttributeError:
+        # then it's a module
+        return obj.__name__.startswith("plenoptic")
+
+
+def is_interesting(obj):
+    # find whether the object is a class or a module
+    is_right_object = inspect.isclass(obj) or inspect.ismodule(obj)
+    return is_right_object and is_part_of_plenoptic(obj)
+
+
+def find_module(obj):
+    members = inspect.getmembers(obj, is_interesting)
+    to_return = []
+    for name, mem in set(members):
+        if inspect.ismodule(mem):
+            # go through the modules recursively
+            to_return.extend(find_module(mem))
+        else:
+            # if they inherit torch Module
+            if torch.nn.Module in mem.__bases__:
```

Comment:
```suggestion
            if issubclass(mem, torch.nn.Module):
```

This would work for nested inheritance too